package.json

```diff
@@ -62,6 +62,7 @@
     "node": ">=10.10.0"
   },
   "dependencies": {
-    "generate-static-index": "file:eng/tools/generate-static-index"
+    "generate-static-index": "file:eng/tools/generate-static-index",
+    "versioning": "file:eng/tools/versioning"
```

Comment:
Why did this need to be changed?

Reply:
So on my local I have to always do the npm install and that adds that path to the dependencies. I can revert that

Reply:
reverted

Reply:
I don't mind it being here I was just not sure what triggered it and not sure why it is needed.